### PR TITLE
clone api for Alarms

### LINKED ISSUE #857: Clone/multi-cluster clone condition to other clusters

### PR DESCRIPTION
**Pull request by alexandreLamarre**
closes #857

### CODE REVIEW

File: plugins/alerting/pkg/alerting/api_conditions.go [hunk 1]
```diff
@@ -574,3 +575,39 @@ func (p *Plugin) Timeline(ctx context.Context, req *alertingv1.TimelineRequest)
...
+	}
+	for _, c := range cl.Items {
+		clusterLookup[c.Id] = struct{}{}
+	}
+	for _, ref := range req.ToClusters {
+		if _, ok := clusterLookup[ref]; !ok {
+			return nil, validation.Errorf("cluster could not be found %s", ref)
+		}
+	}
+	iErrGroup := &independentErrGroup{}
```
Comment (kralicky) [inline plugins/alerting/pkg/alerting/api_conditions.go:597]: Errors are not being added to the error group here?

File: plugins/alerting/pkg/alerting/util.go [hunk 2]
```diff
@@ -75,3 +76,41 @@ func NewCortexAlertingRule(alertId, alertName string, info alertingv1.IndexableM
...
+	defer i.errMu.Unlock()
+	i.errs = append(i.errs, err)
+}
+
+func (i *independentErrGroup) Error() error {
+	if len(i.errs) == 0 {
+		return nil
+	}
+	resErr := ""
+	for _, err := range i.errs {
```
Comment (kralicky) [inline plugins/alerting/pkg/alerting/util.go:109]: Sort and (maybe) dedup first

File: plugins/alerting/pkg/alerting/api_conditions.go (hunk 1 above)
Comment (alexandreLamarre) [reply plugins/alerting/pkg/alerting/api_conditions.go:597]: Added

File: plugins/alerting/pkg/alerting/util.go (hunk 2 above)
Comment (alexandreLamarre) [reply plugins/alerting/pkg/alerting/util.go:111]: Done

**Review by kralicky — APPROVED** [review]

**Pull request merged**